docs/REST-API-Reference.md

```diff
@@ -9,26 +9,14 @@ subject to change at any time. Feedback through our GitHub issues is appreciated
 - [Introduction](#introduction)
 - [Resource Object Schemas](#resource-object-schemas)
 - [API Endpoints](#api-endpoints)
-  - [Bundle Actions API](#bundle-actions-api)
```

Comment:
I'm not sure why this change from https://github.com/codalab/codalab-worksheets/pull/2838 is also in this PR?

Reply:
It's because this PR is stacked on that one. I'm opening a PR per commit.

Reply:
I thought we discussed having multiple PR's with different changes. This makes reviewing more difficult.

Reply:
Oh, you can just go to the last commit of each PR and just review that commit. All previous commits are on other PRs.